src/Geta.Optimizely.Sitemaps/Pages/Geta.Optimizely.Sitemaps/Index.cshtml

```diff
@@ -35,14 +35,15 @@
                     {
                         <tr>
                             <td class="sitemap-name">
-                                @if (Model.ShowHostsLabel)
-                                {
-                                    <label>@Model.HostLabel</label>
-                                }
                                 @if (Model.ShowHostsDropDown)
                                 {
                                     <select asp-for="SitemapViewModel.SiteUrl" asp-items="Model.SiteHosts"></select>
                                 }
+                                else
+                                {
+                                    <label>@Model.HostLabel</label>
+                                    <input type="hidden" class="form-control align-middle" asp-for="SitemapViewModel.SiteUrl" value="@Model.HostLabel"/>
```

Comment:
Do you need CSS class for the hidden input?

Reply:
Haha nope, will remove it